From 548c282ec0146ed6ae2bbdad5518acb08a9c4258 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lu=C3=ADs=20Pereira?= <luis.artur.pereira@gmail.com>
Date: Mon, 2 Feb 2015 11:45:59 -0800
Subject: [PATCH 11/15] Use QPlatformTheme::SystemIconFallbackThemeName in
 static QString fallbackTheme()

Instead of QPlatformTheme::SystemIconThemeName

Port of commit
https://qt.gitorious.org/qt/qtbase/commits/4dae1a685c3d78fa0601e8490117e9a62336c398
---
 qiconfix/qiconloader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qiconfix/qiconloader.cpp b/qiconfix/qiconloader.cpp
index 2c3502a..9519485 100644
--- a/qiconfix/qiconloader.cpp
+++ b/qiconfix/qiconloader.cpp
@@ -75,7 +75,7 @@ static QString fallbackTheme()
 {
 #if 0
     if (const QPlatformTheme *theme = QGuiApplicationPrivate::platformTheme()) {
-        const QVariant themeHint = theme->themeHint(QPlatformTheme::SystemIconThemeName);
+        const QVariant themeHint = theme->themeHint(QPlatformTheme::SystemIconFallbackThemeName);
         if (themeHint.isValid())
             return themeHint.toString();
     }
-- 
1.9.0

